server/mdm/apple/util.go

```diff
@@ -0,0 +1,70 @@
+package apple_mdm
+
+import (
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"encoding/pem"
+	"errors"
+	"fmt"
+)
+
+const rsaKeySize = 2049
```

Comment:
Note that if this is not 2048, Apple will reject the request.

Reply:
I pushed a fix for this.

Reply:
Ty, this was a silly typo